Oleksadr_Petrov/HW4/Task2.py

```diff
@@ -0,0 +1,7 @@
+value = 2
+
+print(2 * 2)
+print(2 << 1)
+
+print(2 // 2)
+print(2 >> 1)
```

Comment:
![image](https://user-images.githubusercontent.com/25909810/188015583-5d7ac718-c0d5-4eb9-9d06-9648144f11d7.png)
This means no empty line at the end of this file